1.train_models/classify_genotypes/nbconverted/classify_genotypes.py

```diff
@@ -2,7 +2,7 @@
 # coding: utf-8
 
 # # Classify WT and Null Genotypes Logistic Regression
-# Plates 3, 3p, 4, and 5 are used in all splits to classify genotypes either (WT or Null)
+# Plates 3, 3p, and 5 are used in all splits to classify genotypes either (WT or Null)
 # The feature selected data is used in all data splits.
 # Pre-evaluation metrics are stored from all splits and these plates.
```

Comment:
It might be useful to describe the manual method of splitting the data here. It should be described somewhere in the end so it might be better in a README potentially.